Sources/GRPC/ClientConnection.swift

```diff
@@ -143,19 +144,24 @@ public class ClientConnection {
   }
 }
 
-// Note: documentation is inherited.
+// MARK: - Unary
+
 extension ClientConnection: GRPCChannel {
-  public func makeUnaryCall<Request: GRPCPayload, Response: GRPCPayload>(
+  private func makeUnaryCall<S: MessageSerializer, D: MessageDeserializer>(
+    serializer: S,
+    deserialiser: D,
```

Comment:
Inconsistent spelling, use `z`.

Reply:
This is pervasive ("find on page" shows 32 instances) so I think you'll need to do a quick search for this in the diff and fix it everywhere.

Reply:
Agh, thanks, good spot.